src/styles/sectionPickerOverrides.less

```diff
@@ -35,6 +35,12 @@
 	color: #604a60;
 }
 
+.SectionPickerContainer {
+	img{ //this is color #604a60 which is color of text in sectionPickerContainer
+		filter: brightness(0) saturate(100%) invert(32%) sepia(7%) saturate(1393%) hue-rotate(251deg) brightness(94%) contrast(96%);
+	}
```

Comment:
Smart, using the color filter generator? :P
I'm guessing this doesn't fix the icon for HC?
This is a good patch for now.

A better fix is to generate the triangle in CSS (rather than img) using https://stackoverflow.com/questions/12483178/how-can-i-make-a-triangle-in-html
This was you can fix the HC color as well.

Reply:
yep, got that. So for making a fuller fix, rather than using filter, converting to font icons and using forced colors for HC would be better here? right? as we have other shapes as well.

Reply:
Yes, but you don't need a new font, UNICODE already has the triangle https://www.bing.com/search?q=unicode+triangle+symbol
Just replace the image with it and use the HC CSS fix.
Also, i'd fix it in the other repo to be a more generic fix

Reply:
The high contrast issue pointed out here will be fixed later as converting the icons will need some time and will be taken up later.